core/standard/abstract_tests/building-blocks/record-collection/ATS_mandatory-properties.adoc

```diff
@@ -3,7 +3,7 @@
 //[width="90%",cols="2,6a"]
 //|===
 //^|*Abstract Test {counter:ats-id}* |*/conf/record-collection/mandatory-properties-collection*
-//^|Test Purpose |Validate the a catalog objects includes all mandatory properties.
+//^|Test Purpose |Validate that a catalog objects includes all mandatory properties.
```

Comment:
This should read "Validate that a catalog object includes all mandatory properties."  That is, remove the "s" from the "objects" word.